micropip/package_manager.py

```diff
@@ -132,6 +136,29 @@ async def install(
             - If a list of URLs is provided, micropip will try each URL in order until \
             it finds a package. If no package is found, an error will be raised.
 
+        extra_index_urls:
```

Comment:
I am not sure if we need to separate `extra_index_urls` as we already support multiple index_urls.

How about exposing an API to get the current index URLs for micropip instead?

For example,

```py
micropip.install("...", extra_index_urls=["A", "B", "C"])
```

can be replaced with

```py
current_index_urls = micropip.get_index_urls()
micropip.install("...", index_urls=current_index_urls + ["A", "B", "C"])
```